kayoumido/HEIGVD-PRO-A-11
# V1/48 delete presentation

this pull request will Close #48 
## How to test it

Make sure to have an authenticated user (with a valid Bearer token) and presentation and presentations owned by yours users.

The delete button is located in the ``/presentation/:id`` page.

## Important (bug)
Du to a bug explained in #162, when issued a delete presentation will not apply change
